src/website/app/demos/Radio/examples/radio-group/basic.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Radio, { RadioGroup } from '../../../../../../Radio';
+import { FormFieldDivider } from '../../../../../../Form';
+import DemoForm from '../../components/DemoForm';
+
+export default {
+  id: 'basic',
+  title: 'Basic Usage',
+  description: `Use either the \`data\` prop or \`children\` to construct a
+group of Radios.
+
+_Note that these basic examples demonstrate uncontrolled components as they have
```

Comment:
This should probably be in a Callout or shortened and put in the description. Consider:

"Use an [uncontrolled]() RadioGroup, by setting the `defaultChecked` prop, when you don't need to maintain the state yourself. The `data` prop and `children` can be used interchangeably"

Reply:
Updated